library/alloc/src/collections/binary_heap.rs

```diff
@@ -1443,6 +1451,22 @@ impl<T: Ord> From<Vec<T>> for BinaryHeap<T> {
     }
 }
 
+#[stable(feature = "std_collections_from_array", since = "1.55.0")]
```

Comment:
I think the stabilized version should've been 1.56 as #87415 has been merged. I'll handle it if this gets merged.